src/services/github-contribution.js

```diff
@@ -0,0 +1,11 @@
+const BASE_URL = 'https://github-contributions-api.now.sh/v1/';
+
+class GithubContribution {
+
+  static getByUsername = username => fetch(`${BASE_URL}${username}`);
+
+  static emptyYears = (years) => (years.length === 0 ? true : false);
```

Comment:
since this is returning a boolean it might be useful to start with `has` like `hasContributions`. `emptyYears` doesn't make sense. Btw, why creating this class?

Reply:
I was thinking in future for more methods, but really, it's unnecessary now, I've removed and fetch again.